lib/rspecq/reporter.rb

```diff
@@ -116,16 +116,18 @@ def humanize_duration(seconds)
     def flaky_jobs_to_sentry(jobs, build_duration)
       return if jobs.empty?
 
-      Raven.capture_message("Flaky jobs detected", level: "warning", extra: {
-        build: @build_id,
-        build_timeout: @timeout,
-        queue: @queue.inspect,
-        object: self.inspect,
-        pid: Process.pid,
-        flaky_jobs: jobs,
-        flaky_jobs_count: jobs.count,
-        build_duration: build_duration
-      })
+      jobs.each do |job|
+        filename = job.sub(/\[.+\]/, '')
+        Raven.capture_message("Flaky test file: #{filename}", level: "warning", extra: {
+          build: @build_id,
+          build_timeout: @timeout,
+          queue: @queue.inspect,
+          object: self.inspect,
+          pid: Process.pid,
+          job_path: job,
+          build_duration: build_duration,
+        }, tags: { 'flaky': true })
```

Comment:
We should also add `spec_file` (filename) as a tag, so that teams can add custom rules and queries based on it.